### ISSUE TITLE
linkerd-viz assumes marathon master runs on localhost

### ISSUE THREAD

**Issue by smparekh** [linkerd/linkerd-viz#27]
In prometheus-mesos-marathon.yml line 11:
```
marathon_sd_configs:
  - servers:
    - 'http://localhost:8080'
```
should be changed to:
```
marathon_sd_configs:
  - servers:
    - 'http://marathon.mesos:8080'
```

To reflect other linkerd mesos-marathon examples.

You can also modify the linkerd-viz.json under mesos-marathon to use add-host parameter if that dns entry will not resolve:
```
{
  "id": "linkerd-viz",
  "instances": 1,
  "cpus": 1.0,
  "mem": 512.0,
  "acceptedResourceRoles": ["*", "slave_public"],
  "maintainer": "hello@buoyant.io",
  "container": {
    "type": "DOCKER",
    "docker": {
      "image": "buoyantio/linkerd-viz:latest",
      "parameters": [
        {
          "key": "add-host",
          "value": "marathon.mesos:192.168.250.11"
        }
      ],
      "forcePullImage": true,
      "network": "HOST",
      "privileged": true
    }
  },
  "args": ["mesos-marathon"],
  ...
```

**Comment by smparekh** [comment]
See PR #28

**Comment by wmorgan** [comment]
Thanks, @smparekh !

**Comment by smparekh** [comment]
There should also be a documentation update on the examples to indicate the change if the mesos master and marathon slave node are on different instances. I just used a random IP but that should be pointed out as a configurable value.

**Comment by siggy** [comment]
Hi @smparekh ! We've just released linkerd-viz 0.1.4, which now supports a `MARATHON_HOST` environment variable. We think this addresses your use case. Let us know if you have any additional questions or comments. Thanks again for filing the issue!
